### ISSUE TITLE
use the deepuq package as the module base to simplify imports

### ISSUE THREAD

**Issue by bnord** [deepskies/DeepUQ#199]
```
!pip install deepuq 
from analyze.analyze import AggregateCheckpoints
from models.models import model_setup_DE
from data.data import DataPreparation
import torch
import numpy as np
import matplotlib.pyplot as plt
from torch.utils.data import TensorDataset, DataLoader
from train import train
```

This code block doesn't work because the system doesn't know where to find `analyze`, `models`, `data`, and `train`. In the readme it says that one can git clone or use pip install. I think updating the base directories so that people can import from deepuq would make this more streamlined. If not more streamlined, I'm not sure people will put in the effort.

**Comment by beckynevin** [comment]
@bnord is this a copy-pasted block of code that you're running in a colab notebook? Does this code existing within a file currently in the repo?

Also can you let me know which version of deepuq you're using?

**Comment by beckynevin** [comment]
Addressed in PR #204 also v0.1.6 of pypi
